packages/block-library/src/page-list/convert-to-links-modal.js

```diff
@@ -4,8 +4,15 @@
 import { Button, Modal } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
-export const convertDescription = __(
-	'This page list is synced with the published pages on your site. Detach the page list to add, delete, or reorder pages yourself.'
+import { createInterpolateElement } from '@wordpress/element';
+
+export const convertDescription = createInterpolateElement(
+	__(
+		"This menu is currently displaying your website's pages. Editing it will allow you to add, delete, or reorder pages, but new pages will no longer be <em>automatically</em> added to this navigation."
```

Comment:
```suggestion
		"This menu is currently displaying your website's pages. Editing it will allow you to add, delete, or reorder pages, but new pages will no longer be <em>automatically</em> added to this menu."
```

Reply:
Perhaps both, "navigation menu".

Reply:
Thanks Alex.

Rich - I think it has to be because we deliberately tried to disambiguate "menu" (navigation menu) and "menu" (restaurant menu).

Don't see any harm in using both terms for extra clarity 👍

Reply:
I just realised we also use "menu" in the first paragraph. We could use `navigation menu` both times. Technically that is what it is...

Reply:
I updated in https://github.com/WordPress/gutenberg/pull/56829/commits/19e7a726859207b9ad78e0045abfee9c200e2f2a.

I will merge sometime tomorrow unless there are any further objections. 

Thank you everyone.